apps/ejabberd/src/ejabberd.app.src

```diff
@@ -40,7 +40,8 @@
                   usec,
                   uuid,
                   xmerl,
-                  worker_pool
+                  worker_pool,
+                  hackney
```

Comment:
I know `fusco` is harder to use and not well documented. Regarding pools `fusco` has this feature as well, take a look at `fusco_cp`. Fusco was create  for and used heavily in load tests (namely  Megaload).

I'm not against `hackney`. If we decided to use it, we should rather change other places from `fusco` to `hackney` (in a separate PR). I just want to avoid multiple libs for the same purpose (if possible).